test/e2e/ginkgo/multiple_cluster_deployment_test.go

```diff
@@ -117,16 +117,16 @@ var _ = ginkgo.Describe("end to end testing", func() {
 			util.ValidateNumDeploymentReplicas(ctx, wec1, ns, 1)
 			util.ValidateNumDeploymentReplicas(ctx, wec2, ns, 1)
 
-			if false { // TODO: enable this after the transport plugin implements create-only
+			if true { // TODO: enable this after the transport plugin implements create-only
```

Comment:
Perhaps should just completely remove.

Reply:
Done. This the only thing changed in the force-push to 08960c7be .